Clew/Controllers/ViewController.swift

```diff
@@ -96,7 +96,7 @@ class ViewController: UIViewController, ARSCNViewDelegate, SRCountdownTimerDeleg
     // MARK: Properties and subview declarations
     
     /// How long to wait (in seconds) between the alignment request and grabbing the transform
-    static let alignmentWaitingPeriod = 5
+    static var alignmentWaitingPeriod = 5
```

Comment:
Looks good! Be aware, although in this case its fine I think, what this line means is that when you modify this value it changes the value for ALL instances of this view controller ('static'). In this case we only have one of them, so I guess it doesn't really matter. Additionally, we use this pattern originally to access this value from child view controllers. Maybe we should change this down the road??

Reply:
if we stop using it as a static, then we have to access the variable via 'self' and 'parent'.